cypress/integration/ui_spec.js

```diff
@@ -382,6 +382,40 @@ describe("UI", () => {
 
 				cy.wait("@createDevice");
 			});
+
+			// TODO: Not sure if we want this or not..
```

Comment:
Lijkt me wel dat we dit willen. Als de API de error `api_key_not_valid` terug geeft zou ik verwachten dat je niet meer de berichten kan zien die je eerder verstuurd hebt.

Reply:
Done